core/src/main/java/jenkins/util/xml/XMLUtils.java

```diff
@@ -37,9 +35,9 @@
 import javax.xml.xpath.XPathFactory;
 
 /**
- * Utilities useful when working with various XML types.
+  * Utilities useful when working with various XML types.
+  * @since 1.596.1 and 1.600, unrestricted since TODO
```

Comment:
`@since` need only mention the version in which it was unrestricted. If anyone cares for some reason about versions in which it was present but restricted (why?), they can use Git history.